src/perf_tools/fiber_trace.cr

```diff
@@ -205,7 +217,7 @@ class Fiber
   end
 
   # crystal-lang/crystal#13701
-  {% if compare_versions(Crystal::VERSION, "1.10.0") < 0 %}
+  {% if compare_versions(Crystal::VERSION, "1.10.0") < 0 && !Fiber.has_constant?(:Stack) && !flag?(:execution_context) %}
```

Comment:
It should be:
```suggestion
  {% unless Fiber.has_method?(:inactive) %}
```

Reply:
Nope that checks for a class methods :cry:

Reply:
I think you mean `Fiber.class.has_method?("inactive")`

Reply:
Oh my, yes, thanks!